tests/boardfarm_plugins/boardfarm_prplmesh/tests/ap_config_renew.py

```diff
@@ -0,0 +1,66 @@
+###############################################################
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+# SPDX-FileCopyrightText: 2020 the prplMesh contributors (see AUTHORS.md)
+# This code is subject to the terms of the BSD+Patent license.
+# See LICENSE file for more details.
+###############################################################
+import time
+
+from .prplmesh_base_test import PrplMeshBaseTest
+from capi import tlv
+
+
+class ApConfigRenew(PrplMeshBaseTest):
+    """Check initial configuration on device."""
+
+    def runTest(self):
+        # Locate agents and controller
+        for dev in self.dev:
+            if dev.controller_entity:
+                controller = dev.controller_entity
+            if dev.agent_entity:
+                agent = dev.agent_entity
+
+        dev.wired_sniffer.start(self.__class__.__name__ + "-" + dev.name)
+        mac_repeater1_upper = agent.mac.upper()
+        controller.ucc_socket.cmd_reply("DEV_RESET_DEFAULT")
+        controller.ucc_socket.cmd_reply(
+            "DEV_SET_CONFIG,"
+            "bss_info1,{} 8x Multi-AP-24G-1 0x0020 0x0008 maprocks1 0 1,"
+            "bss_info2,{} 8x Multi-AP-24G-2 0x0020 0x0008 maprocks2 1 0"
+            .format(mac_repeater1_upper, agent.mac))
+        controller.ucc_socket.dev_send_1905(agent.mac, 0x000A,
+                                            tlv(0x01, 0x0006, "{" + controller.mac + "}"),
+                                            tlv(0x0F, 0x0001, "{0x00}"),
+                                            tlv(0x10, 0x0001, "{0x00}"))
+
+        time.sleep(30)
```

Comment:
Autoconfig renew should finish in a few seconds... Does it really take 30 seconds before it finishes?

Reply:
I wasn't doing a lot of experiments to find a minimal value. 30 seconds is enough for RAX40 to pass the test. Do you think is it worth to invest time and find minimal time required for test to pass on RAX40?

Reply:
Well... This is an actual *sleep*, not just a timeout, so we really loose that time. For this one test it's not a big deal, but if we do something like that in every test, our boardfarm tests are going to take *very long*.

And this really should take only a few hundred milliseconds. If it takes longer than that, we probably have a problem somewhere...